website/docs/getting-started/setup/installation-guides/docker/README.mdx

```diff
@@ -1,203 +1,84 @@
 ---
-description: Appsmith can be deployed locally or on a private instance using Docker
-sidebar_position: 1
+description: Follow these steps to deploy Appsmith on Docker.
 ---
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 # Docker
 
-Docker is an open source [containerization](https://www.ibm.com/in-en/cloud/learn/containerization) platform. It enables developers to package applications into standardized executable components called containers. These containers combine the application's source code with the Operating System (OS) libraries and dependencies required to run that code in any environment.
+This page provides instructions to install Appsmith on Docker.
 
-<VideoEmbed host="youtube" videoId="Tde7GqE6FQQ" /> 
+<VideoEmbed host="youtube" videoId="Tde7GqE6FQQ" title="How to install Appsmith using Docker" 
+caption="How to install Appsmith using Docker Compose"/> 
 
 ## Prerequisites
 
-* [Docker](https://docs.docker.com/get-docker/) (version 20.10.7 or later)
-* [Docker-Compose](https://docs.docker.com/compose/install/) (version 1.29.2 or later)
+1. [Docker](https://docs.docker.com/get-docker/) (version 20.10.7 or later)
+2. [Docker-Compose](https://docs.docker.com/compose/install/) (version 1.29.2 or later)
+3. Create `appsmith` folder
 
-1. Create an installation folder called `appsmith` for deployment and data storage.
-2. `cd` into the installation folder.
+## Install
+Go to the `appsmith` folder and follow the steps below to install Appsmith:
 
-## Setup with Docker Compose _(recommended)_
+<Tabs queryString="current-edition">
+<TabItem label="Community Edition" value="community_edition">
 
-The Appsmith Docker image contains all the components required to run within a single Docker container. All these multiple processes are managed by a [Supervisord](http://supervisord.org/) instance, which is a lightweight process manager.
-
-<Tabs groupId="editions" >
-   <TabItem label="Community Edition" value="community">
-
-1. Click to download the [`docker-compose.yml`](https://bit.ly/docker-compose-CE) file and place it into the Appsmith installation folder.
```

Comment:
@sum35h - Earlier there were two ways in which users could download the `docker-compose.yml`. 
1 - click the link which points to the bit.ly link
2 - Use curl command, which also points to the same bit.ly

As we are asking users to go to the root directory using the `cd` command, users are already on the terminal and would it make more sense to just have the cURL command to download the file?

Please confirm. CC: @sharat87

Reply:
In the new version - I've just kept the cURL command. Please let me know if this makes sense.

Reply:
If this is approved we can go ahead and merge this page

Reply:
Yes, just mentioning the curl command works.